lib/src/settings_section.dart

```diff
@@ -25,12 +25,13 @@ class SettingsSection extends AbstractSection {
   @override
   Widget build(BuildContext context) {
     final platform = Theme.of(context).platform;
-    if (platform.isIOS(context))
+    if (platform.isIOS(context)) {
       return iosSection();
-    else if (platform.isAndroid(context))
-      return androidSection(context);
-    else
+    }
+    if (platform.isAndroid(context)) {
```

Comment:
IMO, the `else` statement makes the code more readable even though it achieves the same result if it's omitted.

Reply:
Funny, but I think the opposite :) I can revert, no problem.